### ISSUE TITLE
Saved models are different when loaded in 

### ISSUE THREAD

**Issue by 11merrettm** [seisbench/seisbench#312]
I have been experimenting with retraining the available models in Seisbench. Once I have trained a model, I am trying to save it to my system for future use. However, when I load my saved models in, I notice that they perform slightly differently, so clearly the model weights, or architecture, are being changed.

To demonstrate, I include some code where I save the pretrained PhaseNet model to my system and then read it in again.

Firstly I save and read in the model as outlined in https://github.com/seisbench/seisbench/pull/71 :
```
pnet = sbm.PhaseNet.from_pretrained("instance")
pnet.save("pnet")
model_copy  = sbm.PhaseNet()
model_loaded = model_copy.load("pnet")
model_loaded.labels =  "PSN"

Plot with:
annotations = pnet.annotate(stream,P_threshold=0.3, S_threshold=0.3 )
annotations2 = model_loaded.annotate(stream,P_threshold=0.3, S_threshold=0.3 )
fig = plt.figure(figsize=(15, 10))
axs = fig.subplots(3, 1, sharex=True, gridspec_kw={'hspace': 0})
offset = annotations[0].stats.starttime - stream[0].stats.starttime
for i in range(3):
    axs[0].plot(stream[i].times(), stream[i].data, label=stream[i].stats.channel)
    if annotations[i].stats.channel[-1] != "N":  # Do not plot noise curve
        axs[1].plot(annotations[i].times() + offset, annotations[i].data, label=annotations[i].stats.channel)
        axs[2].plot(annotations2[i].times() + offset, annotations2[i].data, label=annotations2[i].stats.channel)
axs[0].set_title("Waveform")
axs[1].set_title("pnet")
axs[2].set_title("saved pnet")
axs[0].legend()
axs[1].legend()
```
![image](https://github.com/user-attachments/assets/79ca5230-50b5-4199-a3af-ce8610119030)
Clearly the model behaves differently.

I also try a different approach, by saving and reading in the model as outlined in https://github.com/seisbench/seisbench/issues/69 :
```
pn_model = sbm.PhaseNet.from_pretrained("instance")
torch.save(pn_model.state_dict(), './abc.pt')
model_copy = sbm.PhaseNet()
model_copy.load_state_dict(torch.load('./abc.pt'))
model_copy.labels = "PSN"

annotations = pn_model.annotate(stream,P_threshold=0.3, S_threshold=0.3 )
print(annotations)
annotations2 = model_copy.annotate(stream,P_threshold=0.3, S_threshold=0.3 )
print(annotations2)
fig = plt.figure(figsize=(15, 10))
axs = fig.subplots(3, 1, sharex=True, gridspec_kw={'hspace': 0})
offset = annotations[0].stats.starttime - stream[0].stats.starttime
for i in range(3):
    axs[0].plot(stream[i].times(), stream[i].data, label=stream[i].stats.channel)
    if annotations[i].stats.channel[-1] != "N":  # Do not plot noise curve
        axs[1].plot(annotations[i].times() + offset, annotations[i].data, label=annotations[i].stats.channel)
        axs[2].plot(annotations2[i].times() + offset, annotations2[i].data, label=annotations2[i].stats.channel)
axs[0].set_title("Waveform")
axs[1].set_title("pnet")
axs[2].set_title("saved pnet")
axs[0].legend()
axs[1].legend()
```
![image](https://github.com/user-attachments/assets/5e01b304-2e16-46e0-aa14-a5dd1a4d03ba)

Once again the model I read in behaves differently to the original model. If the model is being correctly saved and read in then I would expect the last two traces to be identical. Am I missing a certain step or overlooking something?  I'd appreciate any pointers.

Thanks,
MM

**Comment by JanisHe** [comment]
Hi @11merrettm,

I tested your code example and found the same error.
However, I think the error only occurs when writing and reading the models with the SeisBench functions for reading and writing models. 
As a quick work around, you can save and load the models with the functions by torch. However, when using torch for saving, you won't have a json file for the saved model. 
So there is a bug when saving the model directly using SeisBench.

I hopes this helps.
Cheers,
Janis

**Comment by yetinam** [comment]
Hi @11merrettm , hi @JanisHe ,
thanks for the detailed issue report and for the assessment. Indeed, there is a bug in the SeisBench wrapper for model saving affecting the metadata. More specifically, the normalization method is not correctly saved, so the loaded model uses a different normalization than the saved model. You can check this by comparing `pnet.norm` to `model_loaded.norm`. I'll write a bugfix and push it later today.

**Comment by yetinam** [comment]
The issue should be fixed in v0.7.3. Please have a look and reopen the issue if the problem persists. Thanks again for the issue report!
